### DISCUSSION TITLE
How to properly adjust the layout colors?

### DISCUSSION THREAD

**Discussion by idanb-droid** [molstar/molstar#795] [Q&A]
I'm trying to adjust the layout colors of the plugin in my React application. While I can change the colors using simple CSS, I always need to add the '!important' rule which I believe is not the best practice. I'm looking for the proper way to adjust the layout colors without using `!important`. 
Has anyone had experience with this issue in a React application with the plugin? Any guidance or suggestions would be appreciated. Thanks in advance!

**Comment by dsehnal** [comment]
I would recommend adjusting directly the SCSS variables. This should be relatively easy by building on the scss https://github.com/molstar/molstar/blob/master/src/mol-plugin-ui/skin/light.scss#L1, which is also included in the NPM package.

You can basically add one more color file akin to https://github.com/molstar/molstar/blob/master/src/mol-plugin-ui/skin/colors/light.scss#L1 and all the other colors should be recomputed accordingly (plus a few more in https://github.com/molstar/molstar/blob/master/src/mol-plugin-ui/skin/base/variables.scss#L1 shared for all the themes).
